### PR TITLE
Modify Makefile in flutter/samples, so that we can run the l10n script

### PR DESCRIPTION
**Pull request by perclasson**
- Remove deploy script as it's no longer used.
- Update the root folder in the Makefile, so that we can run the l10n script.
- Ran the formatter over the localization messages.

### CODE REVIEW

**Comment by perclasson** [comment]
@shihaohong FYI When we are running the formatter for the localization messages it now sorts it in a different order.

File: gallery/gallery/Makefile [hunk 1]
```diff
@@ -33,12 +33,6 @@ master-branch-check: fetch-master
 		$(error Your master branch is not up to date with origin/master, please pull before deploying)
   endif
 
-
-.PHONY: deploy
-deploy: master-branch-check build-web
```
Comment (rami-a) [inline gallery/gallery/Makefile]: You can also remove `master-branch-check` and  `build-web`

File: gallery/gallery/lib/l10n/messages_all.dart [hunk 2]
```diff
@@ -15,485 +15,485 @@ import 'package:intl/intl.dart';
...
-import 'messages_en_AU.dart' as messages_en_au;
-import 'messages_en_CA.dart' as messages_en_ca;
-import 'messages_en_GB.dart' as messages_en_gb;
-import 'messages_en_IE.dart' as messages_en_ie;
-import 'messages_en_IN.dart' as messages_en_in;
-import 'messages_en_NZ.dart' as messages_en_nz;
-import 'messages_en_SG.dart' as messages_en_sg;
-import 'messages_en_US.dart' as messages_en_us;
-import 'messages_en_ZA.dart' as messages_en_za;
+import 'messages_ko.dart' as messages_ko;
```
Comment (rami-a) [inline gallery/gallery/lib/l10n/messages_all.dart]: @shihaohong do you know why the latest version of the l10n script moved all these around?

Comment (shihaohong) [reply gallery/gallery/lib/l10n/messages_all.dart]: This was a regression when I introduced tests, and I made a fix immediately after. 

https://github.com/flutter/flutter/pull/46247

Could you review and it approve it if it looks good?

Comment (perclasson) [reply gallery/gallery/lib/l10n/messages_all.dart]: Thanks. I'll revert the change here then, and then we rerun this when your change has landed in flutter/flutter.

**Comment by rami-a** [comment]
Oh interesting, I was not aware of this package. But I like the idea of replacing our Makefile with it. I'll create a ticket in our backlog for that.

File: gallery/gallery/lib/l10n/messages_all.dart (hunk 2 above)
Comment (shihaohong) [reply gallery/gallery/lib/l10n/messages_all.dart]: The sorting fix has landed! Not sure if I need to let you all know here, but just in case :)

**Pull request merged**